packages/front/src/SingleCardPokemon.tsx

```diff
@@ -0,0 +1,250 @@
+import * as React from 'react';
+import {SingleCardModel, ListModel} from './ModelPokemon';
+import {Card} from 'react-bootstrap';
+import {getSingleCardPokemon} from './conector';
+import { PageEnum} from './EnumTypes';
+
+
+type SinglePokemonS = {
+    pokemon: SingleCardModel,
+    atackMap: string,
+    cartItemsId: Array<ListModel>
+}
+
+export class SingleCardPokemon extends React.Component <any,SinglePokemonS> {
+    constructor(props:any){
+        super(props);
+        this.state= {
+            cartItemsId: [],
+            pokemon: {
```

Comment:
może by to wydzielić jako obiekt i te wartosci ustawiac w konstruktorze?

Reply:
ok